### ISSUE TITLE
[BUG] Shadows HTML -> mPDF

### ISSUE THREAD

**Issue by totoberg123** [mpdf/mpdf#1096]
I have tried CSS 'box-shadow' on a div.

Random HTML tester result: 
![image](https://user-images.githubusercontent.com/1339475/65602185-bf150580-dfa3-11e9-8cdf-71379a61df42.png)

mPDF result: 
![image](https://user-images.githubusercontent.com/1339475/65602262-e23fb500-dfa3-11e9-9c17-360eed949929.png)

As you can see the 'glow' is only shown on the right side of the box.

CSS: 
`box-shadow: 0px 0px 20px 1px grey;`

**Comment by finwe** [comment]
Can you please add a complete PHP/HTML/CSS example as well as versions of PHP and mPDF as required in the issue template?

**Comment by totoberg123** [comment]
I just figured out that the issue is working as it should on Internet Explorer.

On Chrome it does not work.

**Comment by Klap-in** [comment]
What do you mean? that the pdf-viewers in Internet Explorer and in Chrome work differently? What about Acrobat reader?
If you mean something different, please answer the finwe's questions above. That allows to reproduce the case. (For an example, you could have a look at https://github.com/mpdf/mpdf/issues/1088)

**Comment by sml1990** [comment]
Yes, this is also not working on chromium based browsers for me. Firefox is okay. Chromium, same issue as stated above. Definitely a bug.

**Comment by Klap-in** [comment]
Can you please add a complete and minimal PHP/HTML/CSS example as well as versions of PHP and mPDF?

**Comment by sml1990** [comment]
A quick example that will result in the same problem seen in the images above. I've reliably reproduced it in Chromium, but not FF. It seems that others above have been able to reproduce also.

PHP V7.4
mPDF V8.0.7


**CSS:**
.shadow-sm {
  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075) !important;
}

.shadow {
  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.15) !important;
}

.shadow-lg {
  box-shadow: 0 1rem 3rem rgba(0, 0, 0, 0.175) !important;
}

.shadow-none {
  box-shadow: none !important;
}

.bg-success{
  background: green
};

**HTML:**
<div>
<p>Text</p>
<div class="bg-success shadow">
<p>Content</p>
<p>Content</p>
<p>Content</p>
<p>Content</p>
<p>Content</p>
</div>
<p>More text</p>
</div>


**PHP:**
ob_end_clean();
$mpdf = new \Mpdf\Mpdf();
$mpdf->WriteHTML($css, 1);
$mpdf->WriteHTML($html);
$mpdf->Output();
